docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
+
+[[ingest-manager-app-crashes]]
 === The {ingest-manager} app in Kibana crashes.
 
 To find more about the error, open the development console of your browser, navigate to the **Network** tab and refresh the page. One of the requests to the Ingest Manager API will most likely have returned with an error. If the error message doesn't give you enough information to fix the problem on your own, please contact us https://ela.st/ingest-manager-feedback{here}.
 
-=== The endpoint `/api/ingest_management/setup` returns an error that the package registry can't be reached.
 
-In order to install {integrations}, the {ingest-manager} app needs to connect to an external service, the {package-registry}. For this to work, Kibana server must be able to connect to http://epr-experimental.elastic.co on port 80.
\ No newline at end of file
+[[agent-enrollment-timeout]]
+=== I get a time-out when I try to run the agent enrollment on my host
+This can look like the below console output: 
+fail to enroll: fail to execute request to Kibana: Post http://kibana:5601/api/ingest_manager/fleet/agents/enroll?: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+
+This is possibly due to a connectivity problem with the host talking to the Kibana instance.  It may be a networking problem, attempt a 'ping' command from the host to confirm it can reach the Kibana instance.  If this works, then it could be a problem with the specified enrollment command executed.  If 'ping' works, check the URL and port, both are correct for your environment.  If those are correct and working, check the enrollment key in the enrollment command you are trying to execute is valid.  You can do this by viewing the Fleet ui in {ingest-manager} and clicking on the Enrollment Tokens tab.  If you click the 'eyeball' for each listed secret, one listed should match the string being used in your host's enrollment command. If it is not listed, create a new enrollment token and replace it in the command, this can be done in that same UI.
+
+[[enrolled-agent-not-showing-up]]
+=== I ran the Agent enrollment step but I don’t see the Agent in the UI
+If Elastic Agent was successfully enrolled, but doesn't show up in the Fleet list a common problem is that it was not 'started'.  Starting the Agent is a separate step which can be done as follows as the below.
+
+On linux & macos hosts:
+./elastic-agent run
+
+On Windows hosts:
+elastic-agent.exe run
+
+[[where-are-the-agent-logs]]
+=== I need to find the logs when the Agent starts up
+
+When started successfully, the logs are in the folder where the Agent was started, in a sub-directory 'data/logs/metricbeat'
+If that path of logs does not exist, it indicates Agent did not successfully run metricbeat, which is a higher level problem to triage.
+
+[[what-is-my-agent-config]]
+=== I need to know the configuration used in the running Agent.
```

Comment:
This doesn't seem like a troubleshooting topic. Maybe this is something we should explain somewhere else in the docs?